docs/manual_curation.md

```diff
@@ -0,0 +1,126 @@
+# Manual trait curation
+
+
+
+## Extract information about previous mappings
+
+At this step, mappings produced by the pipeline on the previous iteration (including automated and manual) are
+downloaded to be used to aid the manual curation process.
+
+```bash
+# Download the latest eva_clinvar release from FTP
+wget -qO- ftp://ftp.ebi.ac.uk/pub/databases/eva/ClinVar/latest/eva_clinvar.txt \
+  | cut -f4-5 | sort -u > previous_mappings.tsv
+```
+
+
+
+## Create the final table for manual curation
+
+```bash
+python3 bin/trait_mapping/create_table_for_manual_curation.py \
+  --traits-for-curation traits_requiring_curation.tsv \
+  --previous-mappings previous_mappings.tsv \
+  --output table_for_manual_curation.tsv
+```
+
+
+
+## Sort and export to Google Sheets
+
+Note that the number of columns in the output table is limited to 50, because only a few traits have that many
+mappings, and in virtually all cases these mappings are not meaningful. However, having a very large table degrades
+the performance of Google Sheets substantially.
+
+```bash
+cut -f-50 table_for_manual_curation.tsv | sort -t$'\t' -k2,2rn | xclip -selection clipboard
+```
+
+Create a Google Sheets table like this one
+[_traits_requiring_curation_](https://docs.google.com/spreadsheets/d/1mb_ZAEwlSTLCQYBWsihxvUGWoy-otaKFq8tIxpJVT0U/)
+and paste the result to that table.
+
+
+
+## Manual curation criteria
+Good mappings must be eyeballed to ensure they are actually good. Alternative mappings for medium or low quality
+mappings can be searched for using OLS. If a mapping cant be found in EFO, look for a mapping to a HP or ORDO trait
+name. Most HP and ORDO terms will also be in EFO but there are some that arent. These can be imported to EFO using
+the Webulous submission service.
+
+### Criteria to manually evaluate mapping quality
+
+* Exact string for string matches are _good_
+* Slight modifications are _good_ e.g. IRAK4 DEFICIENCY -> Immunodeficiency due to interleukin-1  receptor-associated
+  kinase-4 deficiency
+* Subtype to parent are _good_ e.g ACHROMATOPSIA 3 -> Achromatopsia
+* Parent to subtype are _bad_ e.g. HEMOCHROMATOSIS -> Hemochromatosis type 3
+Familial / congenital represented on only one half are _bad_ e.g. Familial renal glycosuria -> Renal glycosuria
+* Susceptibility on only one half is _bad_ e.g Alcohol dependence, susceptibility to -> alcohol dependence
+* Early / late onset on only one half is _bad_ e.g. Alzheimer disease, early-onset -> Alzheimer's disease
+
+### Unmapped trait names
```

Comment:
Same suggestion as in the section above.

Reply:
Fixed (I presume you meant cant → can't typo fix)

Reply:
That and adding MONDO to the list of ontologies we can get mapping from.

Reply:
Oh that's right! Fixed in https://github.com/EBIvariation/eva-cttv-pipeline/pull/69/commits/f83186727e96e918c5319312f81b770e1a842b3b